glad/documentation.py

```diff
@@ -0,0 +1,171 @@
+from glad.parse import SpecificationDocs, CommandDocs, xml_parse
+from glad.util import prefix, memoize, raw_text
+from shutil import rmtree
+import glad.util
+import subprocess
+import re
+
+
+class DocsGL(SpecificationDocs):
+    URL = 'https://github.com/BSVino/docs.gl.git'
+    CACHED = True # Only clones the DocsGL repo once
+    API = 'gl'
+    docs = dict()
+
+    @property
+    @memoize(method=True)
+    def out_dir(self):
+        from pathlib import Path
+        from tempfile import gettempdir
+        if self.CACHED:
+            return Path('.cached') / 'docs.gl'
+        return Path(gettempdir()) / 'docs.gl'
+
+    def load(self):
+        if self.out_dir.exists() and not self.CACHED:
+            rmtree(str(self.out_dir))
```

Comment:
I feel like the caching behaviour should be passed into the class/load function. Deleting files should be done only on the cli. E.g. this won't behave very nice on the webservice.

Reply:
I remove this caching logic for now, and move it to `__main__.py` similar to how specifications are loaded (from file if exist, otherwise from remote)